dae/dae/studies/study_wrapper.py

```diff
@@ -24,17 +24,18 @@
 
 class StudyWrapper(object):
```

Comment:
this should become GenotypeDataWrapper... and it should be possible to wrap any of GenotypeData instances...